pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "prairielearn"
 version = "0.0.0"
 readme = "README.md"
-requires-python = ">=3.10"
+requires-python = ">=3.10.18,<3.11"
```

Comment:
This is a more correct way to pin, e.g. newer pythons would cause the test suite to fail in `apps/prairielearn/python/test/sympy_utils_test.py`